packages/typespec-test/test/mslearn/generated/typespec-ts/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
```

Comment:
As we already have unit tests for this feature, not sure whether it is necessary add ms-learn as a test project for now.